tools/dtscat

```diff
@@ -46,4 +46,3 @@ do
 done
 
 cat "$BASE" $OVERLAYS
-
```

Comment:
Can you just drop the newline removal and then I'll merge. Thanks

Reply:
Pushed another change to fix that.